src/controllers/article.js

```diff
@@ -342,16 +343,13 @@ export default {
       articleId,
       authorId,
       slug: invalidSlug,
+      id: invalidId,
```

Comment:
InvalidId?, invalidSlug? Could you please explain what's going on here? It seems weird to be updating with an invalid ID and slug

Reply:
> InvalidId?, invalidSlug? Could you please explain what's going on here? It seems weird to be updating with an invalid ID and slug

I discovered that one can decide to break the app by adding the id and slug from the request body. So I decided to remove them by collecting them as invalidId and invalidSlug. This ensures that my slug and id remain immutable.
What the code does is to collect any slug or id passed into the request body and rename them to invalidSlug and invalidId repectively.

Reply:
It's good that you are trying to protect against attacks, but there's really no point in doing this. We are not going to use slug and id anyways so all we need to do is destructure as you have done so that we can get our remaining data i.e. `...article`. I think you needed to rename slug because it was already defined above, this makes sense. You can leave it as it is.